blocks/test/fixtures/core__cover-image.json

```diff
@@ -4,15 +4,30 @@
         "name": "core/cover-image",
         "isValid": true,
         "attributes": {
-            "title": [
-                "Guten Berg!"
-            ],
             "url": "https://cldup.com/uuUqE_dXzy.jpg",
-            "contentAlign": "center",
+            "id": 8398,
             "hasParallax": false,
-            "dimRatio": 40
+            "dimRatio": 50
```

Comment:
It might be nice to keep non-default dim to see the class name included in HTML.